js/apps/system/_admin/aardvark/APP/frontend/js/arango/arango.js

```diff
@@ -1159,18 +1159,25 @@
       });
     },
 
-    getFoxxFlag: function () {
-      var flag;
+    getFoxxFlags: function () {
+      var flags = {};
 
-      if ($('#new-app-replace').prop('checked')) {
-        flag = true;
```

Comment:
Previously the flag was apparently some kind of weird enum. We're now using an object with actual boolean flags.